internal/security/setup/option/optionHelpers.go

```diff
@@ -214,25 +214,48 @@ func markComplete(dirPath string) error {
 	return nil
 }
 
-func getXdgRuntimeDir() string {
+func getWorkDir() string {
 	if xdgRuntimeDir, ok := os.LookupEnv(envXdgRuntimeDir); ok {
-		return xdgRuntimeDir
+		return filepath.Join(xdgRuntimeDir, pkiInitBaseDir)
+	} else if setup.Configuration.SecretsSetup.WorkDir != "" {
+		return setup.Configuration.SecretsSetup.WorkDir
```

Comment:
Configuration should override environment variable auto-detection.

Reply:
Perhaps you should also test that WorkDir exists and is writable.

Reply:
Jim was mentioning that the convention from the CoreWG is that environment variables should override toml settings. Do we want to adopt this convention as well?

Reply:
Yo're right.  I withdraw my comment.